contracts/test/EthEstimateGas.js

```diff
@@ -0,0 +1,92 @@
+const { expect } = require("chai");
+const { ethers, network } = require("hardhat");
+const fetch = require("node-fetch");
+
+const expectedValue = 10;
+const expectedGas = "21000";
+
+let ethCallContract;
+
+const expectThrowsAsync = async (method, errMsgKeyWords) => {
```

Comment:
The `expectThrowsAsync` method is used in multiple files, do you think maybe we should make a util function?

Reply:
Yes it should be extracted to util. Right now this PR has two following PR depending on it, I would like to do a refactor for expectThrowsAsync in another PR, is that good?